src/test/java/com/example/produktapi/selenium/WebShopSeleniumTests.java

```diff
@@ -107,8 +107,29 @@ void checkIfJewelryLinkTextIsDisplayed(){ //Emma
         assertTrue(jewelryLinkText, "Text is not visible");
     }
 
+    @Test
+    void validateCheckoutButtonUpdatedWithURL(){  //Vijaya
+        WebElement checkoutButton = BaseClass.driver.findElement(By.className("btn-warning"));
+        checkoutButton.click();
+        String actualURL = BaseClass.driver.getCurrentUrl();
+        String expectedURL = "https://webshop-agil-testautomatiserare.netlify.app/checkout.html";
+        assertEquals(expectedURL,actualURL);
 
+    }
+    @Test
```

Comment:
What I think there is a bug: When a user searches for a particular product and tries to compare the product title from the search page, it shows the product title from the main page. 
*Anyone can review it?